pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[build-system]
+requires = ["jupyter_packaging~=0.10,<2", "jupyterlab~=3.0"]
```

Comment:
as a pacakger, i much prefer having anything related to dependencies on separate, sorted lines.